pkg/tools/logger/grouplogger.go

```diff
@@ -0,0 +1,99 @@
+package logger
+
+import "context"
+
+type groupLogger struct {
+	loggers []Logger
+}
+
+/*
+func GroupLogFromTypes(ctx context.Context, types []loggerType) Logger {
+	loggers := make([]Logger, len(types))
+	for i, t := range types {
+		if loggers[i] = CreateLogger_CT(ctx, t); loggers[i] == nil {
+			return nil
+		}
+	}
+	return &groupLogger{loggers: loggers}
+}
+*/
+
+func GroupLogFromSlice(ctx context.Context, s []Logger) Logger {
+	loggers := make([]Logger, len(s))
+	for i, t := range s {
+		if loggers[i] = t; loggers[i] == nil {
```

Comment:
Why we check for nil?

Reply:
My intention was returning a nil groupLogger in case one of component logger was nil

Reply:
This is not expected.